README.md

```diff
@@ -254,6 +254,58 @@ auth0.auth
   .catch(console.error);
 ```
 
+#### Login with Passwordless
+
+Passwordless is a two-step authentication flow, and requires the **Passwordless OTP** grant to be enabled in your Auth0 application beforehand. Check [our guide](https://auth0.com/docs/dashboard/guides/applications/update-grant-types) to learn how to enable it.
+
+To start the flow, you request a link or code to be sent to the user's email or phone number:
```

Comment:
You might want to reword this to avoid implying that you can use a link with SMS.